examples/hybrid/gravitywave_parameterization/gravity_wave_parameterization.jl

```diff
@@ -31,17 +78,32 @@ end
 
 function gravity_wave_tendency!(Yₜ, Y, p, t)
     #unpack
-    (;
-        gw_source_height,
-        gw_Bm,
-        gw_F_S0,
-        gw_c,
-        gw_cw,
-        gw_c0,
-        gw_nk,
-        gw_k,
-        gw_k2,
-    ) = p
+    if config == "column"
+        (;
+            gw_source_height,
+            gw_Bm,
+            gw_source_ampl,
+            gw_c,
+            gw_cw,
+            gw_c0,
+            gw_nk,
+            gw_k,
+            gw_k2,
+        ) = p
+    elseif config == "sphere"
+        (;
+            gw_source_pressure,
+            gw_Bm,
+            gw_source_ampl,
+            gw_c,
+            gw_cw,
+            gw_c0,
+            gw_nk,
+            gw_k,
+            gw_k2,
+            params,
+        ) = p
+    end
```

Comment:
How can we reduce the duplication here?

Reply:
Also, can we use dispatch instead of if-else with strings?

Reply:
We can probably just use `gw_source_height` in the column config too, and the cache will be the same variables (although some variables still have different shapes).

Reply:
> We can probably just use `gw_source_height` in the column config too, and the cache will be the same variables (although some variables still have different shapes).

you mean `pressure` right? Yea, using `height` in the single column is only to match the paper results.

Reply:
Haham, yes, I meant `pressure`